src/test/java/com/based/basedsurvey/controller/FillSurveyControllerTest.java

```diff
@@ -139,16 +139,6 @@ public void testFillSurvey() throws Exception {
         String param2 = "values" + oq.getId();
         String param3 = "values" + rq.getId();
 
-        // get the page to fill out the survey and expect it contains all the question prompts
```

Comment:
Test is no longer relevant, it's doesn't make sense to test some changes due to the additions in HTMX